clients/src/main/java/org/apache/kafka/common/requests/SyncGroupResponse.java

```diff
@@ -19,14 +19,12 @@
 import org.apache.kafka.common.protocol.ApiKeys;
 import org.apache.kafka.common.protocol.Errors;
 import org.apache.kafka.common.protocol.ProtoUtils;
-import org.apache.kafka.common.protocol.types.Schema;
 import org.apache.kafka.common.protocol.types.Struct;
 
 import java.nio.ByteBuffer;
 
 public class SyncGroupResponse extends AbstractResponse {
```

Comment:
One benefit of looking at this PR for me is that I am learning about request/responses I never knew about.. Who sends the `SyncGroupRequest`? What exactly is being synced? Do we have docs about what these requests do at a high level? If not, it may be a good habit to start adding Java docs to the top of the classes as they are being modified. This would make the code base more accessible for newbies.